apps/src/applab/dataBrowser/TableControls.jsx

```diff
@@ -51,9 +62,11 @@ const TableControls = React.createClass({
   render() {
     return (
       <div style={styles.container}>
-        <button className="btn" onClick={this.props.addColumn} style={dataStyles.button}>
-          Add column
-        </button>
+        <span style={styles.tableNameWrapper}>
+          <span style={styles.tableName}>
+            {this.props.tableName}&nbsp;
```

Comment:
does `&nbsp` work? i feel like when i've needed something like in the past i had to do something like `{" "}`